File: web/src/model/victron_controller/dashboard/from_0_2_0_command_set-extended-charge-mode.ts
// @ts-nocheck
import {SetExtendedChargeMode as v0_2_0_SetExtendedChargeMode} from './v0_2_0/Command'
import {SetExtendedChargeMode as dashboard_SetExtendedChargeMode} from './Command'

export function convert__command__set_extended_charge_mode__from__0_2_0(from: v0_2_0_SetExtendedChargeMode): dashboard_SetExtendedChargeMode {
    return new dashboard_SetExtendedChargeMode (
        JSON.parse(JSON.stringify(from.value))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_command_set_extended_charge_mode.rs


pub fn convert__command__set_extended_charge_mode__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::command::SetExtendedChargeMode) -> crate::victron_controller::dashboard::command::SetExtendedChargeMode {
    crate::victron_controller::dashboard::command::SetExtendedChargeMode {
        value: serde_json::from_value(serde_json::to_value(&from.value).unwrap()).unwrap(),
    }
}